File: Color_Shooter/desktop/src/com/salvai/centrum/desktop/DesktopLauncher.java
package com.salvai.centrum.desktop;

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application;
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration;
import com.salvai.centrum.CentrumGameClass;

public class DesktopLauncher {
    public static void main(String[] arg) {
        Lwjgl3ApplicationConfiguration config = new Lwjgl3ApplicationConfiguration();
        new Lwjgl3Application(new CentrumGameClass(), config);
    }
}

File: Color_Shooter/desktop/src/com/salvai/centrum/desktop/DesktopLauncher.kt
package com.salvai.centrum.desktop

import com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application
import com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration
import com.salvai.centrum.CentrumGameClass

fun main() {
    val config = Lwjgl3ApplicationConfiguration()
    Lwjgl3Application(CentrumGameClass(), config)
}

